OWML.ModHelper.Menus/ModInputCombinationElement.cs

```diff
@@ -0,0 +1,171 @@
+﻿using OWML.Common;
+using OWML.Common.Menus;
+using OWML.ModHelper.Input;
+using OWML.ModHelper.Events;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+namespace OWML.ModHelper.Menus
+{
+    public class ModInputCombinationElement : ModToggleInput, IModInputCombinationElement
+    {
+        private const float ScaleDown = 0.75f;
+
+        public ILayoutManager Layout { get; private set; }
+
+        override public string Title
+        {
+            get => _combination;
+            set
+            {
+                _combination = value;
+                UpdateContents();
+            }
+        }
+
+        private string _combination;
+        private readonly GameObject _layoutObject;
+        private readonly IModInputHandler _inputHandler;
+
+        private static IModInputCombinationElementMenu _popupMenu;
+
+        public ModInputCombinationElement(TwoButtonToggleElement toggle, IModMenu menu, IModInputCombinationElementMenu popupMenu, IModInputHandler inputHandler, string combination = "") : base(toggle, menu)
+        {
+            _inputHandler = inputHandler;
+            _combination = combination;
+            _layoutObject = toggle.transform.GetChild(1).GetChild(0).GetChild(1).gameObject;
```

Comment:
Is there a safer way to find the Layout? Could easily break in a future update.

Reply:
`gameObject.GetComponentInChildren<SomeBehaviour>(true); // true will include inactive objects`

Reply:
There are other layouts IIRC

Reply:
If it really is not possible to get just using `GetComponent` (maybe with multiple `GetComponent`s?), a slightly better way would be via the object names. Breaks if object names change (which is unlikely anyway), but doesn't break if a new child is added:
`toggle.transform.Find("SomeChild/GrandChild/GreatGrandChild")`.

Adding code like this to OWML seems terrible though. Better to use a different `GetComponent` for each level than GetChild.

Reply:
Alright, now it's findcomponentinchildren.transform.find.findcomponentinchildren
Can't do just two consecutive findcomponentinchildren because it also looks inside parent object